common/src/main/java/feast/common/function/Feature.java

```diff
@@ -14,25 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package feast.serving.util;
+package feast.common.function;
 
-import feast.proto.core.FeatureSetProto.FeatureSetSpec;
-import feast.proto.serving.ServingAPIProto.FeatureReference;
+import feast.proto.serving.ServingAPIProto;
 
-public class RefUtil {
-  public static String generateFeatureStringRef(FeatureReference featureReference) {
+public class Feature {
+
+  /**
+   * Accepts FeatureReference object and returns its reference in String
+   * "project/featureset_name:feature_name".
+   *
+   * @param featureReference
```

Comment:
If you arent going to write a comment then you should link to the type https://stackoverflow.com/questions/5915992/how-to-reference-a-method-in-javadoc

Reply:
You should also talk about required or optional properties and how they will be handled.